ci/vars-functional-test.yml

```diff
@@ -1,6 +1,4 @@
 ---
-# temp: skip os-net-setup
-cifmw_os_net_setup_config: []
```

Comment:
IINM, you said this wasn't resolving the issue, and might be causing other issues. Based on my experience that led to this being added, this is not responsible for this issue.